webpack/components/DashboardSearch.js

```diff
@@ -305,6 +316,17 @@ class DashboardSearch extends SearchStateComponent {
                                     onFocusChange={({ focused }) => this.setState({ focused })}
                                     isOutsideRange={(day) => day.isAfter()}/>
                 </div>
+                <div className="col-md-12 question-form-group">
```

Comment:
I think we need a single source of truth for data collection methods. They are listed in `QuestionEdit` as well. At a minimum, you could pull the select element into its own component that has all of the values.